pkg/services/ngalert/api/api_alertmanager.go

```diff
@@ -86,13 +86,9 @@ func (srv AlertmanagerSrv) RouteGetAlertingConfig(c *models.ReqContext) response
 				secureFields[k] = true
 			}
 			gr := apimodels.GettableGrafanaReceiver{
-				Uid:                   pr.Uid,
```

Comment:
I think we finally need to keep this property and set appropriately [for safely identifying receivers](https://github.com/grafana/grafana/pull/34119#issue-644642260) when updating the configuration.

Reply:
Yeah seems like uid is needed now. But I think it is fine to remove the rest.